frontend/javascripts/messages.js

```diff
@@ -208,6 +208,10 @@ instead. Only enable this option if you understand its effect. All layers will n
   "task.no_tasks_to_download": "There are no tasks available to download.",
   "dataset.upload_success": "The dataset was uploaded successfully.",
   "dataset.upload_failed": "The dataset upload failed.",
+  "dataset.upload_none_zip_error":
+    "It looks like your selected file is not zip file. WebKnossos only supports uploading zip files. Please ensure that your dataset is zipped to an archive.",
```

Comment:
```suggestion
    "It looks like your selected file is not zip file. WebKnossos only supports uploading zip files. Please ensure that your dataset is zipped into an archive.",
```

sounds better imo, but not sure? @fm3 probably knows better :)